src/i18n/locales/zh-CN

```diff
@@ -0,0 +1,375 @@
+      "clear_selection": "取消勾选",
+      "delete_selection": "删除勾选 ({count})",
+      "search": "搜索"
+    },
+    "count": "( {count} / 128 )",
+    "item": {
+      "date": "{date, date, long}",
+      "time": "{date, time, short}"
+    },
+    "photos": {
+      "title": "图片输出",
+      "palette": "调色板",
+      "custom": "自定义",
+      "close": "关闭",
+      "export_selected": "输出选择项",
+      "export_all": "输出所有",
+      "colours": {
+        "gb": "GB",
+        "gb_player": "GB Player",
+        "black_and_white": "黑&白"
+      }
+    }
+  },
+  "saves": {
+    "backups": "备份: {count}",
+    "confirm_delete": "您确认需要移除这个备份目录?",
+    "controls": {
+      "add_backup_location": "添加备份目录",
+      "mister_sync": "MiSTer Sync"
+    },
+    "latest_backup": "最后的备份: {date, date, long}, {date, time, short}",
+    "none": "添加以上备份目录",
+    "not_found": "未找到",
+    "oldest_backup": "最早的备份: {date, date, long}, {date, time, short}",
+    "remove": "移除目录",
+    "view_saves": "浏览存档"
+  },
+  "screenshot_info": {
+    "controls": {
+      "back": "返回列表",
+      "save": "保存",
+      "share": "分享",
+      "upscaled": "画质提升"
+    }
+  },
+  "screenshots": {
+    "controls": {
+      "delete_selected": "删除选择 ({count})",
+      "export_selected": "输出选择 ({count})",
+      "search": "搜索",
+      "select": "选择"
+    },
+    "no_screenshots": "您的截屏都会在此显示"
+  },
+  "settings": {
+    "3d_pocket": {
+      "title": "Pocket颜色:",
+      "black": "黑",
+      "white": "白",
+      "glow": "荧光"
+    },
+    "alternate_skip": {
+      "checkbox": "跳过可选资源",
+      "ramble_1": "This setting will skip processing json files under the `_alternatives` folder,\nmeaning you'll just get the \"main\" titles for the cores which sort their JSON files this way.", //not fully understand
+      "ramble_2": "<b>警告:</b> 关闭此选项可能导致安装时间<b>显著延长</b> ",
+      "title": "跳过可选资源"
+    },
+    "archive": {
+      "save": "保存",
+      "title": "游戏ROM & BIOS 资料库",
+      "warning": "请您自行评估所处地区针对数字版权保护的法律法规风险。\n若您确认使用资料库，请复制下面的链接到文本框并点击 \"保存\"。"
+    },
+    "clear_file_cache": {
+      "button": "清除文件缓存",
+      "ramble": "为确保USB和SD卡资料传输效果，较大的文件会被缓存于您的电脑中，\n如果碰到相关问题，可以尝试清除文件缓存进行修复。",
+      "title": "清除文件缓存"
+    },
+    "disconnect": {
+      "button": "断开连接",
+      "ramble": "返回\"连接到Pocket\" 初始页面 (这一操作并不会卸载弹出系统中的SD卡或USB磁盘)",
+      "title": "断开连接"
+    },
+    "reconnect": {
+      "checkbox": "运行程序时自动重新连接",
+      "ramble": "当程序运行时，会自动尝试读取最后一次使用的目录 <1> { ppath } </1> , 不需要每次点击 \"连接到Pocket\" 按钮",
+      "title": "运行程序时自动重新连接"
+    },
+    "thanks": "鸣谢:"
+  },
+  "time_ago": {
+    "less_than_1_min_ago": "少于一分钟前"
+  },
+  "uninstall_core": {
+    "confirm_text": "这个操作将移除核心文件, 但会保留游戏存档等文件, 您确认卸载吗?",
+    "confirm_title": "卸载核心"
+  },
+  "zip_install": {
+    "cancel_button": "取消",
+    "confirm_button": "确认",
+    "remove_duplicate": "移除已存在的重复文件",
+    "scanning": "扫描文件中...",
+    "moved_files_info": "Handles cases where files have moved, e.g. JSON files are now in `_alternatives`", //not fully understand
```

Comment:
So if core version A has a file `pacman-edition-2.json` then core version B comes out and moves `pacman-edition-2.json` into the `_alternatives` folder this'll notice that's happened and also move the file.

The default behaviour is to not touch existing files & only install / update the files the core wants to install on top, but with this ticked it means the file system on the Pocket is kept cleaner if the core author is clearing up where the instance.json files are